src/lib/dialog/dialog.md

```diff
@@ -126,3 +126,7 @@ that the AOT compiler knows to create the `ComponentFactory` for it.
 })
 export class AppModule() {}
 ```
+
+### Accessibility
+The dialog has `role="dialog". The dialog will focus on the first tabbable element when opened,
```

Comment:
This should mention that the dialog role can be overwritten to `alertdialog` in the appropriate context.